plugin/al/schemas/AL/usd/schemas/maya/tests/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ target_sources(${TARGET_NAME}
 
 target_include_directories(${TARGET_NAME}
     PUBLIC
-        ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../schemas
+        ${PROJECT_SOURCE_DIR}/schemas
```

Comment:
@elrond79 just looking at the dots makes me dizzy! :)

Reply:
Awesome, thanks!